### PR TITLE
Add ability to only show latest of each job/config and choose sort order

### PR DESCRIPTION
**Pull request by Brantone**

### CODE REVIEW

File: app/monitor.js [hunk 1]
```diff
@@ -15,20 +15,30 @@ var async = require('async'),
...
+    sortBuilds = function (newBuilds, sortOrder) {
+        if(sortOrder == "name") {
+            newBuilds.sort(function (a, b) {
+                if(a.project > b.project) return 1;
+                if(a.project < b.project) return -1;
+ 
+                return 0;
+            });
+        }
+        else {
```
Comment (marcells) [inline app/monitor.js]: Please change the sortOrder name to project. So, we can extend it to other build properties in future.

Comment (Brantone) [inline app/monitor.js]: Will do.

**Comment by Brantone** [comment]
Yup, will update Readme.md as well. Wanted to make sure code changes were in the right direction before beefing up docs (ex: things like changing the sortOrder to "project").

**Comment by marcells** [comment]
Thank you! Perfect. After you've updated it, I'll merge it. Including your update to TeamCity.js.

**Comment by Brantone** [comment]
Files updated. Squash changes to single commit (apologies for multiple commits, messed up table for README.md)

File: README.md [hunk 2]
```diff
@@ -64,9 +66,14 @@ The build monitor is configured in the file `config.json` in the app directory.
 ```
 
 In the `monitor` section you can set up some general settings:
-- the update interval (in milliseconds)
-- the number of builds, which will be read and displayed in the web frontend
-- enable or disable some debug output on the console
+
```
Comment (Brantone) [inline README.md]: Changed this to table to match layout in each services config.

**Comment by marcells** [comment]
Let's go. Thanks a lot for the enhancements. 😉

**Comment by Brantone** [comment]
My pleasure 👍

**Pull request merged**